exporter/collector/config.go

```diff
@@ -48,6 +48,17 @@ type ClientConfig struct {
 
 type TraceConfig struct {
 	ClientConfig ClientConfig `mapstructure:",squash"`
+	// AttributeKeyMappings determines how to map from OpenTelemetry attribute keys to
+	// Google Cloud Trace keys.  By default, it changes keys so that they appear more
+	// prominently in the UI.
+	AttributeKeyMappings []AttributeKeyMapping `mapstructure:"attribute_mappings"`
```

Comment:
Nit: can the mapstructure name be consistent with the type and field names? E.g., `AttributeMappings`, `AttributeMapping`, `attribute_mappings`.

Reply:
renamed